src/LayoutManager/LayoutManager.ts

```diff
@@ -216,6 +216,8 @@ export class LayoutManager {
       target.setPositionByOrigin(nextCenter, CENTER, CENTER);
       // invalidate
       target.setCoords();
+      (target.subTargetCheck || target.interactive) &&
+        target.forEachObject((object) => object.setCoords());
```

Comment:
here is a good example of the benefit of separating `setCoords` to `setCornerCoords` and `setControlCoords` because here we need only `setCornerCoords`

Reply:
Do you objections doing that?

Reply:
Also this line is incorrect because it goes down 1 level but should recurse down